examples/flutter_gallery/lib/demo/cupertino/cupertino_alert_demo.dart

```diff
@@ -53,153 +53,148 @@ class _CupertinoAlertDemoState extends State<CupertinoAlertDemo> {
         style: CupertinoTheme.of(context).textTheme.textStyle,
         child: Builder(
           builder: (BuildContext context) {
-            final List<Widget> stackChildren = <Widget>[
-              CupertinoScrollbar(
-                child: ListView(
-                  // Add more padding to the normal safe area.
-                  padding: const EdgeInsets.symmetric(vertical: 24.0, horizontal: 72.0)
-                      + MediaQuery.of(context).padding,
-                  children: <Widget>[
-                    CupertinoButton.filled(
-                      child: const Text('Alert'),
-                      onPressed: () {
-                        showDemoDialog(
-                          context: context,
-                          child: CupertinoAlertDialog(
-                            title: const Text('Discard draft?'),
-                            actions: <Widget>[
-                              CupertinoDialogAction(
-                                child: const Text('Discard'),
-                                isDestructiveAction: true,
-                                onPressed: () {
-                                  Navigator.pop(context, 'Discard');
-                                },
-                              ),
-                              CupertinoDialogAction(
+            return Stack(
```

Comment:
This file was already problematic with the number of nested widgets IMO - it seems more problematic now.

If I were reviewing this as a fresh contribution, I would probably ask the contributor to refactor this into separate widgets so it's more understandable as you go down the heirarchy. I don't think that it's a reason to reject this change, but I wonder if it's always the right thing to do to take a `List<Widget>` and inline it.

Reply:
In this particular case I think the code would be more readable by creating methods for each `onPressed`

Reply:
A goal of UI-as-code was to avoid local variable only present to build list with `for` and `if`.

If it makes sense to have a method to create a widget and then use it in `build` there's no obligation to remove this method to inline its content.